### ISSUE TITLE
changing movie sort method not working

### ISSUE THREAD

**Issue by GoogleCodeExporter** [peteaky/android-xbmcremote#314]
```
When changing the sort method from Title - ascending, nothing happens

What version of XBMC Remote are you using?
0.8.2-beta1

Which Android device are you using (and which ROM, if custom)?
SE Xperia X10 1.6


Which XBMC revision are you running and on which platform?
r31881 Windows
```

Original issue reported on code.google.com by `boczan.t...@gmail.com` on 19 Jul 2010 at 11:39

**Comment by GoogleCodeExporter** [comment]
```
Which section are you browsing?
```

Original comment by `phree...@gmail.com` on 20 Jul 2010 at 7:52

**Comment by GoogleCodeExporter** [comment]
```
Movies tab in the movies section, then pressing options button, then Sort
```

Original comment by `boczan.t...@gmail.com` on 20 Jul 2010 at 9:08

**Comment by GoogleCodeExporter** [comment]
Original comment by `phree...@gmail.com` on 20 Jul 2010 at 9:42
- Changed state: **Accepted**
- Added labels: **Milestone-Release0.8**

**Comment by GoogleCodeExporter** [comment]
```
This issue can be closed as the 0.8.3-beta1 contains the fix.
```

Original comment by `Gabor.Fo...@gmail.com` on 31 Jul 2010 at 12:20

**Comment by GoogleCodeExporter** [comment]
Original comment by `phree...@gmail.com` on 2 Aug 2010 at 7:00
- Changed state: **Fixed**
